src/Orckestra.Composer.Website/App_Data/Razor/Composer/Search/BaseSearchResults.cshtml

```diff
@@ -0,0 +1,195 @@
+                <div class="col-6 d-none d-lg-block">
+                    <div class="pagination-dropdown" v-if="Pagination">
+                        <ul class="pagination m-0  align-items-center justify-content-end">
+                            <li class="page-item previous" v-bind:class="{ disabled: !Pagination.PreviousPage.Url }">
+                                <a class="page-link" v-bind:title="Pagination.PreviousPage.DisplayName" v-on:click="sortingChanged(Pagination.PreviousPage.Url)">
+                                    <i class="fa fa-arrow-circle-left"></i>&#160;{{Pagination.PreviousPage.DisplayName}}
+                                </a>
+                            </li>
+
+                            <li class="page-item mx-2">
+                                <div class="dropdown" v-if="Pagination.CurrentPage">
+                                    <button class="btn btn-outline-secondary  dropdown-toggle" type="button" data-qa="page-select-toggle"
+                                            data-toggle="dropdown" aria-expanded="true">
+                                        @Html.Localize("List-Search", "L_PageInDropDown")&#160;{{Pagination.CurrentPage.DisplayName}}
+                                    </button>
+                                    <div class="dropdown-menu dropdown-menu-sm-right dropdown-menu-md-right" role="menu" data-qa="page-select">
+                                        <a class="dropdown-item" v-for="page in Pagination.Pages" v-on:click="sortingChanged(page.UrlPath)">
+                                            @Html.Localize("List-Search", "L_PageInDropDown")&#160;{{page.DisplayName}}
+                                        </a>
+                                    </div>
+                                </div>
+                            </li>
+
+                            <li class="page-item ">
+                                <span class="d-flex-inline pt-2">
+                                    &#160;@Html.Localized("List-Search", "L_PageOfTotalItems", "{{Pagination.TotalNumberOfPages}}")&#160;
+                                </span>
+                            </li>
+                            <li class="page-item next ml-2" v-bind:class="{ disabled: !Pagination.NextPage.Url }">
+                                <a class="page-link" v-bind:title="Pagination.NextPage.DisplayName" v-on:click="sortingChanged(Pagination.NextPage.Url)">
+                                    {{Pagination.NextPage.DisplayName}}&#160;<i class="fa  fa-arrow-circle-right"></i>
+                                </a>
+                            </li>
+                        </ul>
+                    </div>
+                </div>
+            </div>
+        }
+
+        <div class="products-search-results">
+            <template v-if="TotalCount > 0">
+                <div class="row">
+                    <div v-for="(product, index) in SearchResults" class="col-6  col-md-4  col-xl-3" data-qa="search-results">
```

Comment:
probably it is better use v-bind:key  for iteration elements to avoid some vue issues